lms/djangoapps/mobile_api/courses/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Courses API
+"""
```

Comment:
That implementation definitely makes sense and has some advantages over the current implementation. 
Another point to consider is that as it is structured now the friends/courses returns friends that are filtered by courses they are in. Alternatively the courses/friends returns courses that are filtered based on the friends that are in them. 
Consolidating theses two different return types might be misleading as they return very different types of data. 
Let me know what you think the best way to proceed is.